yt/loaders.py

```diff
@@ -1065,9 +1065,9 @@ def load_unstructured_mesh(
     coordinates,
     node_data=None,
     elem_data=None,
-    length_unit=None,
     bbox=None,
     sim_time=0.0,
+    length_unit=None,
```

Comment:
If you're worried about order consistency, please sort the docstring, not the signature itself. Specifically positional arguments (keyword-only arguments can be safely reordered without breaking backwards compatibility however)